delivery/src/main/java/com/storyous/delivery/common/AutodeclineCountdown.kt

```diff
@@ -0,0 +1,62 @@
+package com.storyous.delivery.common
+
+import android.os.CountDownTimer
+import android.widget.TextView
+import androidx.constraintlayout.widget.Group
+import androidx.core.view.isVisible
+import com.storyous.commonutils.TimestampUtil
+import com.storyous.delivery.common.api.DeliveryOrder
+import kotlin.math.floor
+
+class AutodeclineCountdown(
+    private val textView: TextView,
+    timeToDecline: Long
+) : CountDownTimer(timeToDecline, SECOND_MILLIS) {
+
+    companion object {
+        const val TIME_UNIT = 60f
+        const val SECOND_MILLIS = 1000L
+
+        fun init(deliveryOrder: DeliveryOrder, visibilityGroup: Group, countdownText: TextView): AutodeclineCountdown? {

```

Comment:
přejmenoval bych to na "newInstance()", takhle to vypadá jako bys nastavoval statický konstanty pro všechny

Reply:
Nebo Builder()

Reply:
👍